depthai_ros_driver/test/conversion.cpp

```diff
@@ -0,0 +1,47 @@
+#include <depthai_ros_driver/conversion.hpp>
+
+#include <gtest/gtest.h>
+
+namespace rr {
+bool cvMatAreEqual(const cv::Mat& a, const cv::Mat& b) {
+    if (a.size() != b.size()) {
+        return false;
+    }
+
+    cv::Mat temp;
+    cv::bitwise_xor(a, b, temp);
+    return cv::countNonZero(temp.reshape(1)) == 0;
+}
+
+TEST(chw2hwc, base_comparison) {
+    // test that cvMatAreEqual is working as expected
+    uchar* a = new uchar[3]{1, 2, 3};
+    uchar* b = new uchar[3]{1, 2, 3};
+    uchar* c = new uchar[3]{4, 5, 6};
+
+    cv::Mat A(1, 1, CV_8UC1, a);
```

Comment:
`A(3, 1, ...)`, or `sizeof(a)` instead of `3`.

Reply:
thanks, will fix it